Formula/rust.rb

```diff
@@ -93,8 +87,8 @@ def install
 
     if Hardware::CPU.arm?
       # Fix for 1.49.0-beta, remove when the stable version is released
-      inreplace "src/stage0.txt", "1.48.0", "beta"
-      inreplace "src/stage0.txt", "2020-11-19", "2020-12-23"
+      inreplace "src/stage0.txt", "1.48.0", "1.49.0"
+      inreplace "src/stage0.txt", "2020-11-19", "2020-12-31"
     end
```

Comment:
Is this still needed? The comment says it should be removed.

Reply:
I can't install rust if this block is removed.

```
curl: (22) The requested URL returned error: 404
failed to run: curl -s -y 30 -Y 10 --connect-timeout 30 --retry 3 -Sf -o /private/tmp/xxxxxx.sha256 https://static.rust-lang.org/dist/2020-11-19/rust-std-1.48.0-aarch64-apple-darwin.tar.xz.sha256
Build completed unsuccessfully in 0:00:01
make: *** [all] Error 1
```

Reply:
This is what stage0.txt says:
```
# This file describes the stage0 compiler that's used to then bootstrap the Rust
# compiler itself.
#
# Currently Rust always bootstraps from the previous stable release, and in our
# train model this means that the master branch bootstraps from beta, beta
# bootstraps from current stable, and stable bootstraps from the previous stable
# release.
#
# If you're looking at this file on the master branch, you'll likely see that
# rustc is configured to `beta`, whereas if you're looking at a source tarball
# for a stable release you'll likely see `1.x.0` for rustc, with the previous
# stable release's version number. `date` is the date where the release we're
# bootstrapping off was released.
```

So it is needed until the previous stable version supports ARM. Perhaps just update the comment to say "remove when the 2nd stable ARM version is released"?